example/hadoop/src/main/java/org/apache/iotdb/hadoop/tsfile/TsFileHelper.java

```diff
@@ -132,7 +134,10 @@ public static void main(String[] args) throws FileNotFoundException, IOException
     }
     writeTsFile(filePath);
     TsFileSequenceReader reader = new TsFileSequenceReader(filePath);
-    logger.info("Get file meta data: {}", reader.readFileMetadata());
-    reader.close();
+    try {
+      logger.info("Get file meta data: {}", reader.readFileMetadata());
+    } finally {
+      reader.close();
+    }
```

Comment:
usually sonar recommand to put the closable resource into try()

E.g.

try (TsFileSequenceReader reader = new TsFileSequenceReader(filePath)) {
      logger.info("Get file meta data: {}", reader.readFileMetadata());
}

The resource in try() will be auto closed after run out of try(){}

Reply:
Fixed